.travis.yml

```diff
@@ -107,10 +107,10 @@ install:
   - $JHIPSTER_SCRIPTS/02-generate-project.sh
   - $JHIPSTER_SCRIPTS/03-replace-version-generated-project.sh
 script:
-  - $JHIPSTER_SCRIPTS/03-docker-compose.sh
   - travis_wait $JHIPSTER_SCRIPTS/04-tests.sh
-  - $JHIPSTER_SCRIPTS/05-run.sh
-  - $JHIPSTER_SCRIPTS/06-sonar.sh
+  - $JHIPSTER_SCRIPTS/05-docker-compose.sh
```

Comment:
Be careful when changing order in Travis.
The containers will be launched in background and the application will try to start immediatly after that. So, sometimes, the container is not up and the application will fail because the database didn't have enough time to start.
That's why it's safe to start all these containers before the tests.

Reply:
Ok I `git reset --hard`